File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        hs = set(nums)
        longest = 0
        for n in hs:
            if n-1 not in hs:
                i=0
                while n+i in hs:
                    i+=1
                if i > longest:
                    longest = i
        return longest 



        

File: Data Structures & Algorithms/majority-element/submission-0.cpp
class Solution {
public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int, int> count{};
        int res{}, max{};
        for (int n: nums) {
            count[n]++;
            if (count[n] >= max) {
                res = n;
                max = count[n];
            }
        }
        return res; 
    }
};